crates/forge/src/lib.rs

```diff
@@ -167,32 +38,32 @@ fn replace_id_with_params(
             let fork_target_from_runner_config = runner_config
                 .fork_targets
                 .iter()
-                .find(|fork| fork.name == name)
+                .find(|fork| fork.name() == name)
                 .ok_or_else(|| {
                     anyhow!("Fork configuration named = {name} not found in the Scarb.toml")
                 })?;
 
-            Ok(fork_target_from_runner_config.params.clone())
+            Ok(fork_target_from_runner_config.params().clone())
         }
     }
 }
 
 fn to_runnable(
     compiled_test_crate: CompiledTestCrateRaw,
     runner_config: &RunnerConfig,
-) -> Result<CompiledTestCrateRunnable> {
+) -> Result<TestCrate> {
```

Comment:
Why?

Reply:
The forge-runner now owns the type it runs and it's named like that, so it had to be changed.